content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -54,11 +59,13 @@ rules:
 A `ClusterRole` can be used to grant the same permissions as a `Role`,
 but because they are cluster-scoped, they can also be used to grant access to:
 
-* cluster-scoped resources (like nodes)
-* non-resource endpoints (like "/healthz")
-* namespaced resources (like pods) across all namespaces (needed to run `kubectl get pods --all-namespaces`, for example)
+* cluster-scoped resources (like {{< glossary_tooltip text="nodes" term_id="node" >}})
+* non-resource endpoints (like `/healthz`)
+* namespaced resources (like Pods), across all namespaces.
```

Comment:
prefer using the plural `pods` resource which is what appears in the clusterrole